go/test/endtoend/cluster/mysqlctl_process.go

```diff
@@ -188,6 +189,21 @@ func (mysqlctl *MysqlctlProcess) Stop() (err error) {
 	if err != nil {
 		return err
 	}
+	// We first need to try and kill any associated mysqld_safe process or
+	// else it will immediately restart the mysqld process when we kill it.
+	mspidb, err := exec.Command("sh", "-c",
+		fmt.Sprintf("ps auxww | grep mysqld_safe | grep vt_%010d | awk '{print $2}'", mysqlctl.TabletUID)).Output()
+	if err != nil {
+		return err
+	}
+	mysqldSafePID, err := strconv.Atoi(strings.TrimSpace(string(mspidb)))
+	// If we found a valid associated mysqld_safe process then let's kill
+	// it first.
+	if err == nil && mysqldSafePID > 0 {
+		if err = syscall.Kill(mysqldSafePID, syscall.SIGKILL); err != nil {
+			return err
+		}
+	}
```

Comment:
Shouldn't we return the error returned by `Atoi`?

Reply:
As it is now, it's a best effort attempt to kill mysqld_safe and we don't fail the workflow if we cannot for any reason.

Reply:
that makes sense